rust/stackablectl/src/cmds/release.rs

```diff
@@ -147,6 +181,9 @@ impl ReleaseArgs {
             ReleaseCommands::Describe(args) => describe_cmd(args, cli, release_list).await,
             ReleaseCommands::Install(args) => install_cmd(args, cli, release_list).await,
             ReleaseCommands::Uninstall(args) => uninstall_cmd(args, cli, release_list).await,
+            ReleaseCommands::Upgrade(args) => {
+                upgrade_cmd(args, cli, release_list, &transfer_client).await
+            }
```

Comment:
note (non-blocker): I guess the `Describe` and `Install` should also use the transfer_client. But that can happen later.

Reply:
Maybe there's nothing for them to cache though, so ignore this.